lib/portage/tests/emerge/test_baseline.py

```diff
@@ -38,6 +38,7 @@
 from portage.tests import cnf_etc_path
 from portage.util import ensure_dirs
 from portage.util.futures import asyncio
+from portage.util import movefile
```

Comment:
Our pylint config won't allow an unused import like this. How did the error that you're working around manifest itself?

Reply:
Still not sure what would be the correct solution, unfortunately don't know enough about Portage. I understand this can't be merged in this form.

The errors I got is the following by running `git clean -dfx; pytest "lib/portage/tests/emerge/test_baseline.py::test_portage_baseline[gpkg-binhost emerge]"`.
```
>>> Install dev-libs/B-1 into /tmp/pytest-of-gabor/pytest-199/eprefix0/var/tmp/portage/portage/dev-libs/B-1/image/                                                                    
 * installing something...                                                                                                                                                            
 * ACCESS DENIED:  open_wr:            <...>/portage/lib/portage/util/__pycache__/movefile.cpython-311.pyc.140285464041136                                             
 * ACCESS DENIED:  open_wr:            <...>/portage/lib/portage/util/__pycache__/_xattr.cpython-311.pyc.140285462691888                                               
 * ACCESS DENIED:  open_wr:            <...>/portage/lib/portage/util/__pycache__/movefile.cpython-311.pyc.140557727815200                                             
 * ACCESS DENIED:  open_wr:            <...>/portage/lib/portage/util/__pycache__/_xattr.cpython-311.pyc.140557727825712                                               
 * ACCESS DENIED:  open_wr:            <...>/portage/lib/portage/util/__pycache__/movefile.cpython-311.pyc.140324497282448                                             
 * ACCESS DENIED:  open_wr:            <...>/portage/lib/portage/util/__pycache__/_xattr.cpython-311.pyc.140324497292960                                               
 * called src_install for dev-libs/B-1                                                                                      
```

(`<...>` is edited by me there, it's the absolute path to the directory `portage` is in.)

Reply:
Note: this is an unrelated error you fixed.

Reply:
I can trigger a sandbox violation locally if I run `find lib -name '*.py[co]' -delete` before the tests, but I don't see anything related to `movefile`.

Reply:
I don't understand unfortunately. Maybe it's not that important if it's only on my machine, but I can't verify that, don't understand the cause.

What do you suggest? Remove this `import`, and just merge this without this? I'm fine with that.

Reply:
The sandbox violation can trigger for more than just movefile, so we need to either allow write in sandbox or else pre-compile the pyc/pyo files.

Reply:
This seems like a complicated problem, out of scope. I'll just remove the `import`.

Reply:
This is a script I have in my portage checkout  to compile pyc/pyo files (probably inspired by [python-utils-r1.eclass](https://github.com/gentoo/gentoo/blob/master/eclass/python-utils-r1.eclass) usage of compileall in its python_optimize function):
```bash
#!/usr/bin/env bash

[[ $UID != 0 ]] && \
	find lib -uid 0 -name "*.py[co]" -print0 | xargs -r -0 rm -f
python -c 'import compileall ; compileall.compile_dir("./lib/", quiet=True)'
python -O -c 'import compileall ; compileall.compile_dir("./lib/", quiet=True)'
```